src/ASIFT_Master_Deliverable_1.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Nov 20 12:24:43 2018
+
+ASIFT_Master_Deliverable_1.py -- Runs the master algorithm, using each phase code as needed.
+for Deliverable 1, take two images, run ASIFT against each of them, and generate matching keypoints as a CSV file.
+If the target image is geo-located, output the geo-location (lat/lon) of each keypoint as well.
+
+@author: mmacferrin
+"""
+import argparse
+
+# TODO: UPDATE THESE IMPORTS TO REFLECT LATEST CODE CHANGES.
+# Import sub-directory phases
+import PHASE_1_IMAGE_PREPROCESSING.ASIFT_Tile_Images as TILING
+import PHASE_2_KEYPOINT_GENERATION.ASIFT_Scheduler as ASIFT
+import PHASE_3_RANSAC_FILTERING.ASIFT_RANSAC_filter as RANSAC
```

Comment:
@mmacferrin  Hello Mike, 

I think we should convert these to lowercase as well.

Reply:
Agreed, and consistent with the comments below, the code would be better named "process_two_images.py" which describes what it does. I hadn't touched that code in awhile, and forgot to update it with the new directory names in mind.